pkg/util/abiutil/abiutil_test.go

```diff
@@ -23,3 +24,58 @@ func TestMustParseABI(t *testing.T) {
 
 	MustParseABI("invalid abi")
 }
+
+func TestUnpackBigInt(t *testing.T) {
```

Comment:
This function name should be named `TestUnpack`, leaving out `BigInt` part since we are testing all `UnpackSomething` functions here.

Could all testes related to one method be in one subtest or at least close to each other?